tests/console/python3_pynacl.pm

```diff
@@ -0,0 +1,77 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Simple pynacl libsodium smoke test with transactional and non-transactional support
+#
+# Maintainer: QE Core <qe-core@suse.de>
+#
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use serial_terminal 'select_serial_terminal';
+use registration qw(runtime_registration);
+use transactional qw(trup_call process_reboot);
+use version_utils;
+use python_version_utils;
+
+my $requires_scc_registration = is_sle_micro || is_sle;
+my $script_download_path = "~/test_pynacl.py";
+
+sub run_test {
+    my ($python_package) = @_;    # Proper declaration of $python_package
+    my $pkg = "$python_package-PyNaCl";
+
+    # we don't run the test if PyNaCl is not packaged for this python version
+    return unless script_run("zypper se $pkg") == 0;
```

Comment:
If we're scheduling in 15-SP4+ we risk missing the package not being shipped/removed without us knowing, ideally we'd like to flag this from QE